MAUI/Calendar/calendar-identifier.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Calendar identifier in .NET MAUI Calendar control | Syncfusion
+description: Learn here all about the Calendar identifier in Syncfusion .NET MAUI Calendar (SfCalendar) control and more details.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Calendar identifier in .NET MAUI Calendar (SfCalendar)
+The .NET MAUI Calendar supports Calendar Identifier to display the different types of calendars based on various cultures and regions. The supported identifies are Gregorian, Hijri, Korean, Persian, Taiwan, ThaiBuddhist, and UmAlQura. Change the calendar identifier by using the [Identifier] property in the SfCalendar.
```

Comment:
The .NET MAUI Calendar supports Calendar Identifier to display the different types of calendars based on various cultures and regions. The supported identifies are Gregorian, Hijri, Korean, Persian, Taiwan, ThaiBuddhist, and UmAlQura. Change the calendar identifier using the [Identifier] property in the SfCalendar.

Reply:
Content review updated